dojo/tools/acunetix360/parser.py

```diff
@@ -0,0 +1,88 @@
+import json
+import re
+
+from dojo.models import Finding, Endpoint
+
+__author__ = "Muhammed Kılıç"
+__status__ = "Development"
+
+
+# Function to remove HTML tags
+TAG_RE = re.compile(r'<[^>]+>')
+
+
+def cleantags(text=''):
+    prepared_text = text if text else ''
+    return TAG_RE.sub('', prepared_text)
+
+
+class Acunetix360Parser(object):
+
+    def get_scan_types(self):
+        return ["Acunetix360 Scan"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return "Acunetix360 Scan"
+
+    def get_description_for_scan_types(self, scan_type):
+        return "Acunetix360 JSON format."
+
+    def get_findings(self, filename, test):
+        tree = filename.read()
+        try:
+            data = json.loads(str(tree, 'utf-8-sig'))
+        except:
+            data = json.loads(tree)
+        dupes = dict()
+
+        for item in data["Vulnerabilities"]:
+            mitigation = ''
+            impact = ''
+            references = ''
+            findingdetail = ''
+            title = ''
+            request = ''
+            response = ''
+
+            title = item["Name"]
+            findingdetail = cleantags(item["Description"])
+            cwe = int(item["Classification"]["Cwe"]) if "Cwe" in item["Classification"] else None
+            sev = item["Severity"]
+            if sev not in ['Info', 'Low', 'Medium', 'High', 'Critical']:
+                sev = 'Info'
+            mitigation = cleantags(item["RemedialProcedure"])
+            references = cleantags(item["RemedyReferences"])
+            url = item["Url"]
+            impact = cleantags(item["Impact"])
+            dupe_key = title
+            request = item["HttpRequest"]["Content"]
+            if request is None or request.lenght <= 0:
+                request = "Request Not Found"
+            response = item["HttpResponse"]["Content"]
+            if response is None or response.length <= 0:
+                response = "Response Not Found"
+
+            finding = Finding(title=title,
+                              test=test,
+                              description=findingdetail,
+                              severity=sev.title(),
+                              mitigation=mitigation,
+                              impact=impact,
+                              references=references,
+                              cwe=cwe,
+                              static_finding=True)
+
```

Comment:
There is a date in the reports, you can use it to set `date` attribute.

Reply:
I am not sure about formatting so would be grateful if you help us with using `item["Generated"]` value @damiencarol

Reply:
I was thinking to use the something like that:
```python
datetime.datetime.strptime(obj["Target"]["Initiated"][0:10], "%d/%m/%Y")
```

Reply:
Used phrase from qualys and result is --> ``` scan_date = datetime.datetime.strptime(item["Generated"], "%Y-%m-%dT%H:%M:%SZ").date() ```